src/libraries/System.Private.CoreLib/src/System/Collections/Generic/List.cs

```diff
@@ -426,7 +426,7 @@ public void CopyTo(T[] array, int arrayIndex)
         /// <summary>
         /// Ensures that the capacity of this list is at least the specified <paramref name="capacity"/>.
         /// If the current capacity of the list is less than specified <paramref name="capacity"/>,
-        /// the capacity is increased by continuously twice current capacity until it is at least the specified <paramref name="capacity"/>.
+        /// the capacity is increased to at least twice the current capacity, and to at least <paramref name="capacity"/>.
```

Comment:
From a documentation perspective, I think it would suffice to say that the capacity will be increased so that it is greater than or equal the specified capacity. The growth function is an implementation detail and subject to change.

Reply:
Personally I prefer being precise here, because it's pretty unexpected that the capacity should double when you asked to ensure `Capacity + 1` (say), but I am not invested in that.